langlib/lang.query/src/main/ballerina/helpers.bal

```diff
@@ -73,6 +73,11 @@ function createSelectFunction(function(_Frame _frame) returns _Frame|error? sele
     return new _SelectFunction(selectFunc);
 }
 
+function createOnConflictFunction(function(_Frame _frame) returns _Frame|error? onConflictFunc)
```

Comment:
This function can convert into a arrow function

Reply:
I have followed the existing coding pattern used in this file. @MaryamZi wdyt? Should I update it? Or should I send a separate PR updating all the places?

Reply:
We can follow best practices for new code (especially given that these are internal functions) and fix/update the older code separately in a separate PR. I wouldn't consider changing old code with this particular change as a high-priority task though.